.github/ISSUE_TEMPLATE.md

```diff
@@ -1,53 +1,41 @@
-<!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
+<!-- Thanks for filing an issue! Please follow this template: 
+This form is for bug reports and feature requests ONLY! 
 
-**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/.):
+If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
+-->
 
-**Note:** Please file issues for subcomponents under the appropriate repo
+**Is this a BUG REPORT or FEATURE REQUEST?**:
 
-| Component | Repo                                                               |
-| --------- | ------------------------------------------------------------------ |
-| kubectl   | [kubernetes/kubectl](https://github.com/kubernetes/kubectl/issues/new) |
-| kubeadm   | [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm/issues/new) |
+<!-- Uncomment only one, leave it on its own line: 
 
-**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.):
+/kind bug
+/kind feature
 
----
+--> 
 
-**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
+** What SIG should look at this?**:
 
-<!--
-If this is a BUG REPORT, please:
-  - Fill in as much of the template below as you can.  If you leave out
-    information, we can't help you as well.
+<!-- Uncomment which SIG:
+https://github.com/kubernetes/kubernetes/blob/master/labels.yaml#L255-L302
+Syntax is `/sig aws` as an example, multiple SIGs are allowed
 
-If this is a FEATURE REQUEST, please:
-  - Describe *in detail* the feature/behavior/change you'd like to see.
+/sig name-of-sig-goes-here
```

Comment:
I think this will trigger the bot to complain about nonexistent SIGs-- it doesn't parse the <!-- comment syntax.